HealthyLLM.xcodeproj/project.pbxproj

```diff
@@ -534,7 +544,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"HealthyLLM/Supporting Files/Preview Content\"";
-				DEVELOPMENT_TEAM = CQRZ4E7K9U;
```

Comment:
I will revert the development team and bundle identifier after addressing all other PR comments